student.rb

```diff
@@ -4,6 +4,7 @@ class Student < Person
   attr_accessor :classroom
 
   def initialize(classroom)
+    super()
```

Comment:
- [ ] Good job so far 👍 in the initialize method, kindly add all the parameters needed to create a `student` object which are age, classroom, name = "Unknown", parent_permission = true.
- [ ] also when calling the super(), you've to add the parameters gotten from the parent class. Therefore, you should have something like this:   

```ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```